components/gearScript/zen/fn_zen_createSupplyCrate_performAction.sqf

```diff
@@ -25,5 +25,5 @@ if (isNull _unit) then
 else
 {
     ["Replacing inventory of '%1' with contents of '%2'.", _unit, _chosenGear] call zen_common_fnc_showMessage;
-    [_chosenGear, _unit, _chosenFaction] call f_fnc_assignGear;
+    [_chosenGear, _unit, _chosenFaction] remoteExecCall ["f_fnc_assignGear", _unit];
```

Comment:
A little more efficient to use `remoteExec` because `f_fnc_assignGear` uses the `RUN_AS_ASYNC` macro at the top, so it will force itself to use the scheduled environment anyway.
```suggestion
    [_chosenGear, _unit, _chosenFaction] remoteExec ["f_fnc_assignGear", _unit];
```